public/js/util/source-map.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
```

Comment:
> this will leak sourcemaps across navigations (and potentially load wrong sourcemaps if ids are the same across navigations of different sites).

That's a good point. I'll add it to the tracking ticket.

To summarize, we could keep the maps in the store and access the cached consumers/nodes from that.